docusaurus.config.js

```diff
@@ -152,7 +152,7 @@ module.exports = {
               banner: 'unreleased',
             },
             'stable': {
-              label: 'v1.0.2 (stable)',
```

Comment:
we want to depreciate `1.0.2` now - check into archiving it it

Reply:
actually - revert the changes to the zh translations: these are going to be a bit behind for now unfortunately